Content.Server/Clothing/Systems/ChameleonClothingSystem.cs

```diff
@@ -49,6 +58,21 @@ private void OnSelected(EntityUid uid, ChameleonClothingComponent component, Cha
         SetSelectedPrototype(uid, args.SelectedId, component: component);
     }
 
+    private void OnEmpPulse(EntityUid uid, ChameleonClothingComponent component, ref EmpPulseEvent args)
+    {
+        if (component.EmpAffected)
+        {
+            if (component.EmpContinious)
+                component.NextEmpChange = _timing.CurTime + TimeSpan.FromMilliseconds(MathF.Floor(1000 / component.EmpChangeIntensity));
```

Comment:
doing the math in milliseconds is kinda weird when you could just do this:
```suggestion
                component.NextEmpChange = _timing.CurTime + TimeSpan.FromSeconds(1f / component.EmpChangeIntensity);
```

i especially don't get the purpose of the `Floor` operation